core/src/main/java/org/apache/iceberg/ManifestFilterManager.java

```diff
@@ -154,6 +164,12 @@ void caseSensitive(boolean newCaseSensitive) {
   void delete(F file) {
     Preconditions.checkNotNull(file, "Cannot delete file: null");
     invalidateFilteredCache();
+    if (file.manifestLocation() == null) {
+      this.allDeletesReferenceManifests = false;
+    } else {
+      manifestsReferencedForDeletes.add(file.manifestLocation());
+    }
+
```

Comment:
Minor: Let's either add an empty line before `if` or remove this one for consistency.

Reply:
Done!